documentation/synthetic_tokens/tutorials/liquidation.md

```diff
@@ -0,0 +1,423 @@
+
+The dispute bot monitors all liquidations occurring within a given expiring multi-party contract and initiates disputes against liquidations it deems invalid, as inferred from off-chain information about the value of the price identifier.
+A liquidation is invalid if a position was in fact overcollateralized at the time of liquidation.
+
+## Implementation
+
+The liquidation and dispute bots are separate entities. Each has its own wallet and is designed to be be run independently of the other.
+This decouples dependencies between the bots to decrease the risk of one crashing impacting the other.
+The bots are written in Javascript and re-use much of the upstream codebase.
+In production, it is suggested to run the bots within docker containers to further isolate the bots from any systemic risk and to ensure a reproducible execution environment.
+
+## Technical Tutorial
+
+This tutorial will be broken down into three main sections:
+
+1. [Running the bots directly from your host environment (no Docker) from the command line](#running-the-liquidator-and-disputer-bots-locally)
+2. [Running the bots within a dockerized environment from the official UMA Docker image](#running-the-bots-locally-with-Docker)
+3. [Deploying bots to production in Google Cloud Compute](#running-the-bots-in-the-cloud-with-GCP)
+
+This tutorial will guide you through setting up a liquidator and disputer bot to monitor an expiring multi party deployed on the Kovan test network.
+A verified version of the expiring multi party contract can be found on Kovan [here](https://kovan.etherscan.io/address/0xDe15ae6E8CAA2fDa906b1621cF0F7296Aa79d9f1).
+This contract is an ETHBTC synthetic, collateralized in Dai.
+
+## Prerequisites
+
+### Funding accounts
+
+Bots must be funded with currency to pay for liquidations and disputes.
+Specifically, if you want to run a liquidator bot, you need to fund the wallet with 1) synthetic tokens to close liquidated positions, 2) collateral currency to pay the DVM final fee, and 3) Ether to pay transaction fees.
+If you want to run a disputer bot, then this wallet should be funded with 1) collateral currency to pay dispute bonds and the DVM final fee and 2) Ether to pay for transactions.
+
+### Bot private key management
+
+All deployment configurations require a wallet mnemonic (or private key) to be injected into the bots enabling them to submit transactions to preform on-chain liquidations and disputes.
+You can either bring your own mnemonic from an existing wallet or generate a fresh one using the `bip39` package installed within the UMA repo.
+If you have a wallet mnemonic already you can skip this section.
+
+To generate a new mnemonic you can run the following from the `/core` directory:
+
+```bash
+# Start the truffle console
+truffle console --network kovan_mnemonic
```

Comment:
nits:

1. I think we should probably put the `npm install` instructions above this section so they already have node, npm, and the bip39 lib installed, right?
2.  We probably don't need to use truffle here, right. A single line command you could use would be:
```
node -e "console.log(require('bip39').generateMnemonic())"
```

I know it's a little messier, but it makes it a lot quicker IMO. Thoughts?

Reply:
bump

Reply:
I agree on both points. Will update.